services/galley/schema/src/V56_ConferenceCallingPaymentStatus.hs

```diff
@@ -0,0 +1,33 @@
+  -- This file is part of the Wire Server implementation.
+--
+-- Copyright (C) 2020 Wire Swiss GmbH <opensource@wire.com>
+--
+-- This program is free software: you can redistribute it and/or modify it under
+-- the terms of the GNU Affero General Public License as published by the Free
+-- Software Foundation, either version 3 of the License, or (at your option) any
+-- later version.
+--
+-- This program is distributed in the hope that it will be useful, but WITHOUT
+-- ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+-- FOR A PARTICULAR PURPOSE. See the GNU Affero General Public License for more
+-- details.
+--
+-- You should have received a copy of the GNU Affero General Public License along
+-- with this program. If not, see <https://www.gnu.org/licenses/>.
+
+module V56_ConferenceCallingPaymentStatus
+  ( migration,
+  )
+where
+
+import Cassandra.Schema
+import Imports
+import Text.RawString.QQ
+
+migration :: Migration
+migration = Migration 56 "Add payment status config for conference calling" $ do
```

Comment:
Please combine the statements of 55 and 56 into a single migration step, if introduced in one and the same PR. That can be done in the same CQL statement as here you're operating on the same table; or in general also in multiple CQL statements but of the same migration, see e.g. https://github.com/wireapp/wire-server/blob/develop/services/brig/schema/src/V30.hs.

Reply:
It's two different PRs, but this one is based on #1937, which hasn't been merged yet.